BlogTemplate/Pages/Post.cshtml.cs

```diff
@@ -21,6 +30,12 @@ public PostModel(BlogDataStore dataStore)
 
         public Post Post { get; set; }
 
+        public HtmlString HtmlBody()
+        {
+            var html = Markdown.ToHtml(Post.Body, pipeline);

```

Comment:
The markdown rendering could be refactored into a service, so that it could be reused between the post body here and possibly in other places too (such as comments at some point?)